src/components/Sections/hooks/useKonnectorsByCat.ts

```diff
@@ -104,33 +76,41 @@ export const useKonnectorsByCat = (): Section[] => {
       : []
   }, [suggestedKonnectorsQuery.data, candidatesSlugBlacklist])
 
+  const [groupedData, setGroupedData] =
+    useState<{ [key: string]: IOCozyKonnector[] }>()
+
   useEffect(() => {
     const fetchData = async (): Promise<void> => {
       if (!client) return
       const grouped = await fetchAllKonnectors(client)
```

Comment:
I missed this last time but why don't you use a useQuery here? Is there a caveat that you are try to avoid ?

Reply:
Well it's not really a query but a fetch to the registry. It seems I can only use fetchjson, is it not?

Reply:
This point is outdated we need to improve the AppRegistryCollection to be able to remove this useEffect be a useQuery 🙁 Maybe next time